.github/workflows/mingw_w64/start.sh

```diff
@@ -64,7 +64,7 @@ WINE_SYSDIR=$WINE_PREFIX/drive_c/windows
 wine64 cmd /c dir
 ln -s $MINGW_PREFIX/libstdc++-6.dll $WINE_SYSDIR
 ln -s $MINGW_PREFIX/libgcc_s_seh-1.dll $WINE_SYSDIR
-ln -s $MINGW_PREFIX/libgcc_s_sjlj-1.dll $WINE_SYSDIR
+# ln -s $MINGW_PREFIX/libgcc_s_sjlj-1.dll $WINE_SYSDIR
```

Comment:
im curious/clueless why this change? not familiar with how windows is done

Reply:
It was commented-out since it is unavailable from `gcc-mingw-w64-x86-64` (this was probably up until 18.04). These are [exception handling methods](https://stackoverflow.com/questions/15670169/what-is-difference-between-sjlj-vs-dwarf-vs-seh), and this one is for setjmp/longjmp.